pkg/noderesourcetopology/types.go

```diff
@@ -0,0 +1,19 @@
+package noderesourcetopology
```

Comment:
As mentioned above, this doesn't look necessary to be extracted into a single file.